[documentation]: Review language-specific guides (#70)
hyperledger/iroha-2-docs: src/guide/rust.md

This creates an asset `time` that can only be minted once and has the type
`fixed`. `AssetDefinition::fixed` just like its other cousins (`quantity`
and `big_quantity`) returns a builder of an `AssetDefinition`. If you get

> odd error messages, that's because you didn't call `build()` at the end.

Author: I think this part is unnecessary due to #53

};
```

> Needless to say that a synchronous infinite blocking loop is bad UX for

Reviewer: ```suggestion
Needless to say, an synchronous infinite blocking loop is bad UX for
```
